### DISCUSSION TITLE
Remove/disable Cascade on start

### DISCUSSION THREAD

**Discussion by Jamesstevens-3rd** [valinet/ExplorerPatcher#1763] [Ideas]
would be a great if we could turn off the default cascading of windows explorer on start

fyi if i recall correctly this "feature" was first implemented through an update to Internet Explorer v5.5 back in 1999

**Comment by pyrates999** [comment]
Can you show a picture of what you mean?

**Reply by Jamesstevens-3rd** [reply]
https://answers.microsoft.com/en-us/windows/forum/all/how-to-prevent-windows-from-cascading-on-open/c5bcb22d-ad8b-4374-a8da-1e96a73c1111

**Comment by pyrates999** [comment]
I've been able to recreate it by opening file explorer more then once.  Unsure of how to change this though.

**Comment by pyrates999** [comment]
Here's a link to someone suggesting how to enable and disable it: https://www.windowsphoneinfo.com/threads/how-to-prevent-windows-from-cascading-on-open.677173/

**Reply by Jamesstevens-3rd** [reply]
no, that's just how to use the feature cascade which in my opinion is a pretty useless feature.

this does not stop explorer from cascading on start up, this behaviour was first introduced by an Internet Explorer update after 5.5

**Reply by pyrates999** [reply]
If Microsoft fixed it in windows 11, I would try to petition them to fix it in windows 10.  But since windows 10 is now just on maintenance, I don't think they would change this.

**Reply by Jamesstevens-3rd** [reply]
its not fixed in windows 11 

win key + e  will cascade

**Reply by pyrates999** [reply]
If you knew what dll it was happening in, perhaps explorer patcher could change the setting.

**Reply by Jamesstevens-3rd** [reply]
sorry for the late reply, work commitments = boring lol

when I I bought a new laptop in 2014 it came with windows 8.1 and for the most part it was ok but I have always loathed The Ribbon which MS forced on the File Explorer so I used to edit **ExplorerFrame.dll** (system32 and sysWOW64 versions) with **Resourcer** to delete it.

I remember MS introduced it in a Internet Explorer update not long after IE5.5 (IE5.5 was the last non cascading version) and that has always made me wonder if that's why we still have IE11 installed by default throughout windows XP - 11 even though we cant use it any more.

**Reply by pyrates999** [reply]
Cascading for new windows makes sense to me because then you can see the multiple windows easily.
